clang/test/CIR/CodeGen/vector.cpp

```diff
@@ -96,3 +96,36 @@ void foo2(vi4 p) {}
 
 // OGCG: %[[VEC_A:.*]] = alloca <4 x i32>, align 16
 // OGCG: store <4 x i32> %{{.*}}, ptr %[[VEC_A]], align 16
+
+void foo3() {
+  vi4 a = { 1, 2, 3, 4 };
+  int e = a[1];
+}
```

Comment:
Can you add a test where the index of the element being extracted is a variable?